forms-shared/src/example-forms/getExampleFormPairs.ts

```diff
@@ -11,18 +14,25 @@ type ExampleFormPair = {
 
 export function getExampleFormPairs({
   formDefinitionFilterFn = () => true,
+  devForms = false,
 }: {
   formDefinitionFilterFn?: (formDefinition: FormDefinition) => boolean
+  devForms?: boolean
 } = {}): ExampleFormPair[] {
-  const slugs = Object.keys(exampleForms)
+  const getFormDefinitionBySlugLocal = devForms
+    ? getFormDefinitionBySlugDev
+    : getFormDefinitionBySlug
+  const exampleFormsLocal = devForms ? exampleDevForms : exampleForms
```

Comment:
If we have `devForms: true`, shouldn't we be browsing `[...exampleDevForms, ...exampleForms]` or something like that?